test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/RazorPages_DesignTime.codegen.cs

```diff
@@ -29,23 +34,17 @@ public class TestFiles_IntegrationTests_CodeGenerationIntegrationTest_RazorPages
         }
         #pragma warning restore 219
         private static System.Object __o = null;
-        private global::DivTagHelper __DivTagHelper = null;
         #pragma warning disable 1998
         public async override global::System.Threading.Tasks.Task ExecuteAsync()
         {
-            __DivTagHelper = CreateTagHelper<global::DivTagHelper>();
-#line 29 "TestFiles/IntegrationTests/CodeGenerationIntegrationTest/RazorPages.cshtml"
-                                         __o = Name;
+#line 30 "TestFiles/IntegrationTests/CodeGenerationIntegrationTest/RazorPages.cshtml"
+                                         __o = Model.Name;
 
 #line default
 #line hidden
-            __DivTagHelper = CreateTagHelper<global::DivTagHelper>();
-            __DivTagHelper = CreateTagHelper<global::DivTagHelper>();
-            __DivTagHelper = CreateTagHelper<global::DivTagHelper>();
```

Comment:
Look over all your test changes, it looks like TagHelpers are falling off the radar everywhere.